sys/include/psa_crypto/psa/crypto_sizes.h

```diff
@@ -543,7 +546,7 @@ extern "C" {
  *
  *          See also @ref PSA_CIPHER_FINISH_OUTPUT_SIZE().
  */
-#define PSA_CIPHER_FINISH_OUTPUT_MAX_SIZE 0 /* implementation-defined value */
+#define PSA_CIPHER_FINISH_OUTPUT_MAX_SIZE 64 /* implementation-defined value */
```

Comment:
```suggestion
#define PSA_CIPHER_FINISH_OUTPUT_MAX_SIZE 64
```

those were only place-holders for the empty defines